apps/demo-bun-fuels/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "private": true,
+  "name": "demo-bun-fuels",
+  "description": "Simple demo using Bun with Fuels CLI",
+  "author": "Fuel Labs <contact@fuel.sh> (https://fuel.network/)",
+  "scripts": {
+    "pretest": "bun fuels build"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@fuel-ts/account": "workspace:^",
+    "@fuel-ts/errors": "workspace:^",
+    "fuels": "workspace:*"
+  },
+  "version": null,
+  "devDependencies": {
+    "bun": "^1.1.9"
```

Comment:
With this, we don't need `bun` installed, correct?

I can't decide if this is good or bad. I like it tho. 🙂